keras_cv/models/segmentation/segment_anything/sam_layers.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.api_export import keras_cv_export
+from keras_cv.backend import keras
+from keras_cv.layers.serializable_sequential import SerializableSequential
+
+
+@keras_cv_export("keras_cv.layers.MLP")
+class MLP(keras.layers.Layer):
```

Comment:
This is a public class here - it should probably be a private subclass, especially since there was an MLP with the same name in a layer related to this, iirc

Reply:
Done. I have removed the export.